src/components/focusManager.js

```diff
@@ -38,13 +44,22 @@ define(['dom'], function (dom) {
 
     function focus(element) {
 
+        var currentScroll = { x: window.scrollX, y: window.scrollY };
+
         try {
             element.focus({
-                preventScroll: false
+                preventScroll: true
             });
         } catch (err) {
             console.log('Error in focusManager.autoFocus: ' + err);
         }
+
+        // Restore scrolling - Tizen does not support 'preventScroll' option for 'focus'
+        if (browser.tizen) {
+            window.scroll(currentScroll.x, currentScroll.y);
+        }
+
+        scrollManager.scrollIntoView(element);
```

Comment:
It seems that Tizen 4.0 does not support smooth scrolling and ignores 'preventScroll' option. Instant scroll on focus looks ugly. So, I wrote scroll manager to handle all scrolling tasks, set 'preventScroll' to true again and block scrolling by reverting window scroll for Tizen.

Reply:
Did you check that it doesn't prevent scrolling on non-Tizen then?

Reply:
ScrollManger was not tested on non-Tizen devices.
Originally it was in the Tizen app itself (adapter) while I played with it on TV. Keyboard navigation and scroll on focus was in the Tizen app too. Then jellyfin-web got keyboardnavigation and scroll on focus (preventScroll=false), that broke my app. Since I have to make a lot of changes to jellyfin-web, I decided to integrate scrollManager and some code of Tizen app to jellyfin-web and share it with other TVs (as I thought) to have the same scroll behavior.
Is keyboard navigation expected in the browser (not app) on the TV?

Reply:
Ideally web UI should work mostly the same on a browser opened in a TV, plus keyboard navigation on non-TVs should also work (eventually, not sure if it works okay now - I think we had reports that it behaves strangely).

Reply:
Have tested ScrollManager in browser on TV (Samsung UE65NU7400) - works same way (not taking into account loosing of the focus due to missing my app wrapper).

Also have installed webOS TV emulator. Have tested in jellyfin-webos app (it is almost "clear" browser, i.e. with no extra code) and have found out that webOS also ignores 'preventScroll=true'. Have blocked scroll with window.scroll as well, and ScrollManager started to work same way.

ScrollManager has an issue with scrolling of itemcontainer (for example, at "Movies/Recommended" page), possibly because I scroll the window but not the container itself. One way is to wrap all single row containers with emby-scroller (it is "self-scrolled"). Other is to try to scroll parent container with ScrollManager.

I began to split the changes into smaller parts, testing them separately.